contracts/base/LoansBase.sol

```diff
@@ -193,16 +171,9 @@ contract LoansBase is Base {
             .div(DAYS_PER_YEAR_4DP);
 
         // check that enough collateral has been provided for this loan
-        uint256 collateralNeededToken = _getCollateralNeededInTokens(
-            _getTotalOwed(loanID),
-            loans[loanID].loanTerms.collateralRatio
-        );
-        uint256 collateralNeededWei = _convertTokenToWei(collateralNeededToken);
+        (, , , bool requireCollateral) = _getCollateralInfo(loanID);
 
-        require(
-            loans[loanID].collateral >= collateralNeededWei,
-            "MORE_COLLATERAL_REQUIRED"
-        );
+        require(!requireCollateral, "MORE_COLLATERAL_REQUIRED");
```

Comment:
I think `requireCollateral` is a bad variable name, because `require(!requireCollateral` reads 
"require not require collateral" which is a weird statement

I think that "moreCollateralRequired" makes more sense (like your revert message)